etna/transforms/timestamp/holiday.py

```diff
@@ -7,14 +8,43 @@
 import numpy as np
 import pandas as pd
 
+from etna.datasets import TSDataset
 from etna.transforms.base import IrreversibleTransform
 
 
+def compare_frequency(freq: str):
+    """Compare frequency with day."""
+    new_freq = freq
+    secs = 0
+    seconds_in_day = 8640
+    while new_freq != "":
+        numbers_part = "".join(itertools.takewhile(str.isdigit, new_freq))
+        new_freq = new_freq[len(numbers_part) :]
+        numbers_part = "1" if numbers_part == "" else numbers_part
+        letters_part = ""
+        for char in new_freq:
+            if char.isupper():
+                letters_part += char
+            else:
+                break
+        new_freq = new_freq[len(letters_part) + 1 :]
+        if letters_part == "S":
+            secs += int(numbers_part)
+        elif letters_part == "MIN":
+            secs += int(numbers_part) * 60
+        elif letters_part == "H":
+            secs = int(numbers_part) * 360
+        else:
+            return False
+    return secs <= seconds_in_day
+
+
```

Comment:
What do you think about this solution?
```python
def bigger_than_day(freq): 
    dt = '2000-01-01'
    dates_day = pd.date_range(start=dt, periods=2, freq="D")
    dates_freq = pd.date_range(start=dt, periods=2, freq=freq)
    return dates_freq[-1] > dates_day[-1]
```
It seems more elegant and more robust. I think, we should use something similar.